api/src/main/java/io/opentelemetry/api/baggage/EntryMetadata.java

```diff
@@ -26,6 +26,9 @@
    * @return an {@code EntryMetadata}.
    */
   public static EntryMetadata create(String metadata) {
```

Comment:
Mark metadata as `Nullable`

Reply:
We don't use Nullable on APIs which shouldn't be passed null but we have defensive behavior anyways. IIRC this is the answer to all your comments?